general/overlay/usr/sbin/extutils

```diff
@@ -21,6 +24,14 @@ case "${CMD}" in
 		echo "The ipctool installed as remote GitHub plugin"
 	fi
 	$IPCTOOL $@
+  ;;
+  check_mac)
+	if [ "$(fw_printenv -n ethaddr)" = "00:00:23:34:45:66" ]; then
+		status="The eth0 interface has a lousy MAC, please change it.."
+	else
+		status="The eth0 interface has a correct MAC - $(fw_printenv -n ethaddr)"
+	fi
+    echo ${status}
```

Comment:
If `$status` is not used elsewhere, just drop the assignment and use two `echo` instead.

Reply:
As i known, yaml-cli doesn't work with arrays, maybe need to add this functional.